File: src/tests/Markify.Fixtures/Projects/Source/Enum/EnumSamples.vb
Public Enum FooType
End Enum

Public Partial Class ParentType
    Public Enum NestedType
    End Enum
End Class

Public Partial Class ParentType
    Public Partial Class AnotherNestedType
        Public Enum DeeperNestedType
        End Enum
    End Class
End Class

Namespace FooNamespace
    Public Enum InNamespaceType
    End Enum
End Namespace

Namespace FooNamespace.BarNamespace
    Public Partial Class ParentType
        public Enum NestedType
        End Enum
    End Class
End Namespace

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Enum/EnumSamples.cs
public enum FooType { }

public partial class ParentType
{
    public enum NestedType { }
}

public partial class ParentType
{
    public partial class AnotherNestedType
    {
        public enum DeeperNestedType { }
    }
}

namespace FooNamespace
{
    public enum InNamespaceType { }
}

namespace FooNamespace.BarNamespace
{
    public partial class ParentType
    {
        public enum NestedType { }
    }
}